### PR TITLE
update allergyIntolerance with new status param, new examples

### PR DESCRIPTION
**Pull request by daliboz**
Updated params. Added a required field (will be pushing this through to other resources). Thoughts on doing this this way?

Updated examples.

### CODE REVIEW

File: lib/resources.rb [hunk 1]
```diff
@@ -1297,50 +1297,83 @@ def get_resource(key)
...
+                      "relation"=> "self",
+                      "url"=> "https=>//fhir.sandboxcernerpowerchart.com/may2015/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f/AllergyIntolerance?patient=2744010"
+                  }],
+        "entry"=> [{
+                       "resource"=> {
+                           "resourceType"=> "AllergyIntolerance",
+                           "id"=> "3643731",
+                           "meta"=> {
+                               "versionId"=> "3643735",
+                               "lastUpdated"=> "2015-03-11T20=>34=>59.000Z"
```
Comment (MaxPhilips) [inline lib/resources.rb]: Switch back to colons, lines 1303, 1307, 1321, 1332, 1349, 1355, 1366 too

File: lib/resources.rb [hunk 2]
```diff
@@ -1297,50 +1297,83 @@ def get_resource(key)
...
+                           },
+                           "text"=> {
+                               "status"=> "generated",
+                               "div"=> "&lt;div>&lt;p>&lt;b>Allergy Intolerance&lt;/b>&lt;/p>&lt;p>&lt;b>Patient&lt;/b>: RHEUM, TEST ONE&lt;/p>&lt;p>&lt;b>Allergy&lt;/b>: Penicillin&lt;/p>&lt;p>&lt;b>Status&lt;/b>: Refuted&lt;/p>&lt;p>&lt;b>Criticality&lt;/b>: &lt;/p>&lt;p>&lt;b>Category&lt;/b>: Medication&lt;/p>&lt;p>&lt;b>Reactions&lt;/b>: &lt;/p>&lt;p>&lt;b>Onset&lt;/b>: &lt;/p>&lt;p>&lt;b>Comment&lt;/b>: &lt;/p>&lt;/div>"
```
Comment (MaxPhilips) [inline lib/resources.rb]: Looks fine on the site itself but do we want to either write the &lt;'s as <'s or the >'s as &gt;'s?

**Comment by MaxPhilips** [comment]
I think the required column is definitely helpful.  Although it looks a little strange to have the Y/N's as normal text while the entries in Name and Type are monospaced.  Does it make sense to monospace the Required? entries just to keep things looking similar?

**Comment by MaxPhilips** [comment]
+1 after resources.rb updates

File: lib/resources.rb (hunk 2 above)
Comment (daliboz) [inline lib/resources.rb]: I kept only the < as &lt; so that it wouldn't display the text as raw html

**Comment by daliboz** [comment]
Corrected hash rocket issues.

File: lib/resources.rb [hunk 3]
```diff
@@ -1297,50 +1297,83 @@ def get_resource(key)
...
+                               "lastUpdated"=> "2015-03-11T20:34:59.000Z"
+                           },
+                           "text"=> {
+                               "status"=> "generated",
+                               "div"=> "&lt;div>&lt;p>&lt;b>Allergy Intolerance&lt;/b>&lt;/p>&lt;p>&lt;b>Patient&lt;/b>: RHEUM, TEST ONE&lt;/p>&lt;p>&lt;b>Allergy&lt;/b>: Penicillin&lt;/p>&lt;p>&lt;b>Status&lt;/b>: Refuted&lt;/p>&lt;p>&lt;b>Criticality&lt;/b>: &lt;/p>&lt;p>&lt;b>Category&lt;/b>: Medication&lt;/p>&lt;p>&lt;b>Reactions&lt;/b>: &lt;/p>&lt;p>&lt;b>Onset&lt;/b>: &lt;/p>&lt;p>&lt;b>Comment&lt;/b>: &lt;/p>&lt;/div>"
+                           },
+                           "recordedDate"=> "2015-03-11T15:34:59.000",
+                           "recorder"=> {
+                               "reference"=> "Practitioner/3270007",
+                               "display"=> "Venkat, Savitha"
```
Comment (dennispatterson) [inline lib/resources.rb]: I realize that her name will come back when the url is queried, but any concern over publishing Savitha's name on here?

**Comment by jfrhong** [comment]
+1

**Comment by daliboz** [comment]
Updated the display name to be more example-like.

**Comment by dennispatterson** [comment]
+tardis

**Pull request merged**